packages/ai-mcp/src/node/mcp-server.ts

```diff
@@ -13,8 +13,20 @@
 //
 // SPDX-License-Identifier: EPL-2.0 OR GPL-2.0-only WITH Classpath-exception-2.0
 // *****************************************************************************
-import { StdioClientTransport } from '@modelcontextprotocol/sdk/client/stdio';
-import { Client } from '@modelcontextprotocol/sdk/client/index.js';
+import {StdioClientTransport} from '@modelcontextprotocol/sdk/client/stdio';
+import {SSEClientTransport} from '@modelcontextprotocol/sdk/client/sse';
+import {Client} from '@modelcontextprotocol/sdk/client/index.js';
+
+export type TransportType = 'stdio' | 'sse';
+
+export interface MCPServerConfig {
+    name: string;
+    command?: string;
+    args?: string[];
+    env?: { [key: string]: string };
+    transportType?: TransportType;
```

Comment:
Having the transport Type and the sseUrl as separate variables is maybe not optimal, if you use SSE as a transport type, the url is mandatory. As we currently support two transport types, I would either
1. Only provide the seeURL and if it is set we use SSE
2. Include the transaport specific parameters (env and URL) into a transportType object (with two versions)
The second options seems cleaner to me, as env is ignored for SSE. This way, the API makes it very clear what parameters you can provide